src/Stripe.net/Services/Sources/SourceMandateOfflineOptions.cs

```diff
@@ -0,0 +1,12 @@
+namespace Stripe
```

Comment:
This and the other one should be `SourceMandateAcceptanceOfflineOptions` instead.